dark/diamond/sam.py

```diff
@@ -160,15 +159,17 @@ def _SAMLine(self, match, protein, genome):
         qseqid = (match['qseqid'] if self._keepDescriptions else
                   match['qseqid'].split(None, 1)[0])
 
-        ranges = GenomeRanges(protein['offsets'])
+        # from dark.genbank import GenomeRanges
+        # ranges = GenomeRanges(protein['offsets'])
 
         # matchStartInGenome = ranges.startInGenome(match)
         # queryStartInGenome = matchStartInGenome - match['qstart'] - 1
         # orientations = ranges.orientations()
 
-        print('ranges', ranges)
-        print('protein', protein)
-        print('match', match)
+        # print('ranges', ranges)
+        # print('protein', protein)
+        # print('match', match)
+
```

Comment:
Can't ^^ lines just go?

Reply:
I want to leave them in as that code isn't finished. There are some `raise NotImplementedError` lines in there as well.